roles/validate/files/defaults.yml

```diff
@@ -20,7 +20,7 @@
 # SPDX-License-Identifier: MIT
 
 ---
-defaults:
+factory_defaults:
```

Comment:
@mtarking may have figured out a way to allow us to specify the custom defaults under `host_vars` instead of using the ENV variable.  Can you make the following update to your schema file and see if you can get the import to work from `host_vars` instead?

```yaml
---
vxlan: include('vxlan', required=False)
defaults: any(required=False) 
```

If this does work then any `custom defaults` will automatically get defined under a var `defaults` when this [task](https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/validate/tasks/sub_main.yml#L63) is executed.

This would then need to be merged with custom defaults directly after this [task](https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/validate/tasks/sub_main.yml#L63)

Reply:
Fixed.